### ISSUE TITLE
replace the supertest mechanic in the discord bot

### ISSUE THREAD

**Issue by sbenthall** [yrevocnu/psychedelic-glass-bead-game#16]
The supertest mechanic in the discord bot is an outrage.
OUTRAGE!

**Comment by sbenthall** [comment]
What would it be better to replace this with @benheller ?

**Comment by sbenthall** [comment]
I'd like to tackle this.
I'm wondering if there's a standard place to include helper functions in an Express directory/filesystem hierarchy.

I see:

```
app/
    models/
    routes/
    services/
        discord/
```
I'm thinking after reading a few blog posts about it that maybe `discord/` should be popped up to the same level as `routes/` and the handlers moved to `services`. But I defer to you.

**Comment by benheller** [comment]
I agree that `discord/` should be at the same level as `routes/`. I'd move the handlers to a new folder at `app/controllers` or `app/handlers` that's shared between `app/routes/` and `app/discord/`. If you need other helper functions (for things like array manipulation, sorting, etc) those could go in a new `app/utils` folder.

**Comment by sbenthall** [comment]
Thank you @benheller !
I will prioritize this ticket.
